pkg/controllers/scheduledevent/scheduledeventcontroller.go

```diff
@@ -434,6 +432,38 @@ func (s ScheduledEventController) provisionScheduledEvent(templates *hfv1.Virtua
 		}
 	}
 
+	err := s.createAccessCode(se)
```

Comment:
gets displayed a little bit wrong by github.
What i have done is factored out the "createAccessCode" Function si ot can be reused. Creation of accessCode is just done by a new Function.